### DISCUSSION TITLE
NoSQL Database at Oracle CloudWorld 2022

### DISCUSSION THREAD

**Discussion by dario-vega** [oracle/nosql#48] [General]
Oracle [CloudWorld 2022](https://www.oracle.com/cloudworld/) is almost here, and there are [more than few ways](https://www.oracle.com/cloudworld/learning/) to immerse yourself in learning or acquiring skills and knowledge. Our attendees can personalize their learning path and choose various ways to engage and interact with industry peers and Oracle technology experts.
In this [blog post](https://blogs.oracle.com/nosql/post/get-insights-on-building-data-driven-applications-fast-with-nosql-database-at-oracle-cloudworld-2022), We would like to highlight few [Oracle NoSQL Database](https://www.oracle.com/database/nosql/) sessions that you should consider attending ...

https://user-images.githubusercontent.com/66302636/194312281-4789e71c-1f48-49ac-8564-f5cc05e3f401.mp4

**Comment by mikebrey** [comment]
Come on over to our demo booth and have deep-dive discussions with development.
